localdtorage.js

```diff
@@ -0,0 +1,16 @@
+const contactForm = document.forms[0];
+const formDetails = {};
+
+if (window.localStorage.getItem('data')) {
+  const parsedData = JSON.parse(window.localStorage.getItem('data'));
+  contactForm.name.value = parsedData.name;
+  contactForm.email.value = parsedData.email;
+  contactForm.message.value = parsedData.message;
+}
+function collectFormData() {
+  formDetails.name = contactForm.name.value;
+  formDetails.email = contactForm.email.value;
+  formDetails.message = contactForm.message.value;
```

Comment:
- [ ] [OPTIONAL] The form data for all inputs should be stored in a single object and then saved to a single local storage registry, you can save the data directly on formDetails, so your code is simpler
e.g 
```
const formData = {
    name: nameInput.value,
    email: emailInput.value,
    body: bodyInput.value,
};
```

Reply:
@Ismailco I will do this